sklearn/feature_extraction/tests/test_text.py

```diff
@@ -832,6 +832,20 @@ def test_vectorizer_min_df():
     assert len(vect.stop_words_) == 5
 
 
+@pytest.mark.parametrize(
+    "min_df, max_df, message",
+    (
+        (None, 2.0, "max_df == 2.0, must be <= 1.0."),
+        (1.5, None, "min_df == 1.5, must be <= 1.0."),
+    ),
+)
+def test_vectorizer_max_df_unwanted_float(min_df, max_df, message):
```

Comment:
Since you as well modify `max_features`, could you add the validation in this check?

Reply:
You can rename the test `test_vectorizer_params_validation`